src/main/java/org/spongepowered/common/mixin/core/world/MixinGameRules.java

```diff
@@ -24,20 +24,43 @@
  */
 package org.spongepowered.common.mixin.core.world;
 
-import com.google.common.collect.Sets;
 import net.minecraft.world.GameRules;
+
+import org.spongepowered.api.world.gamerule.DefaultGameRules;
 import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.common.interfaces.world.IMixinGameRules;
 
+import java.util.HashSet;
 import java.util.Set;
 import java.util.TreeMap;
 
 @Mixin(GameRules.class)
 public abstract class MixinGameRules implements IMixinGameRules {
 
-    private static final Set<String> DEFAULT = Sets.newHashSet((new GameRules()).getRules());
+    private static final Set<String> DEFAULT = new HashSet<String>();
+
+    static {
```

Comment:
@Cybermaxke : Okay thank you so much 
https://github.com/SpongePowered/SpongeCommon/pull/1021/commits/1ceb19d64e3e69c1bb41081c8c9d866076ec2ce1